src/db/file.rs

```diff
@@ -260,20 +254,6 @@ pub fn add_path_into_database<P: AsRef<Path>>(conn: &Connection,
     file_list_to_json(file_list_with_mimes)
 }
 
-#[cfg(not(windows))]
-fn load_mime_data() -> Result<Cookie> {
-    let cookie = Cookie::open(flags::MIME_TYPE)?;
-    cookie.load::<&str>(&[])?;
-    Ok(cookie)
-}
-
-#[cfg(not(windows))]
-fn detect_mime(content: &Vec<u8>, file_path: &Path, cookie: &Cookie) -> Result<String> {
-    let mime = cookie.buffer(&content)?;
-    correct_mime(&mime, &file_path)
-}
-
-#[cfg(windows)]
 fn detect_mime(_content: &Vec<u8>, file_path: &Path) -> Result<String> {
```

Comment:
Since we don't need the contents anymore in any of these, could you drop that argument as well?